### PR TITLE
Tweak/typography default fonts as util

### PR DESCRIPTION
**Pull request by rotemee**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: includes/utils.php [hunk 1]
```diff
@@ -566,4 +566,14 @@ public static function parse_custom_attributes( $attributes_string, $delimiter =
 
 		return $result;
 	}
+
+	public static function get_default_fonts() {
```
Comment (matipojo) [inline includes/utils.php:570]: Move to `kits_manager`.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by kobizz — DISMISSED** [review]

File: includes/controls/groups/typography.php [hunk 2]
```diff
@@ -88,17 +86,11 @@ public static function get_type() {
...
-		if ( $default_fonts ) {
-			$default_fonts = ', ' . $default_fonts;
-		}
-
 		$fields['font_family'] = [
 			'label' => _x( 'Family', 'Typography Control', 'elementor' ),
 			'type' => Controls_Manager::FONT,
 			'default' => '',
-			'selector_value' => 'font-family: "{{VALUE}}"' . $default_fonts . ';',
+			'selector_value' => 'font-family: "{{VALUE}}"' . Plugin::$instance->kits_manager->get_default_fonts() . ';',
```
Comment (matipojo) [inline includes/controls/groups/typography.php:93]: Move the comma from `get_default_fonts` to here.
```suggestion
			'selector_value' => 'font-family: "{{VALUE}}, "' . Plugin::$instance->kits_manager->get_default_fonts() . ';',
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by rodolphebertozzo — DISMISSED** [review]

File: includes/controls/groups/typography.php [hunk 3]
```diff
@@ -88,17 +86,11 @@ public static function get_type() {
...
-		if ( $default_fonts ) {
-			$default_fonts = ', ' . $default_fonts;
-		}
-
 		$fields['font_family'] = [
 			'label' => _x( 'Family', 'Typography Control', 'elementor' ),
 			'type' => Controls_Manager::FONT,
 			'default' => '',
-			'selector_value' => 'font-family: "{{VALUE}}"' . $default_fonts . ';',
+			'selector_value' => 'font-family: "{{VALUE}}", ' . Plugin::$instance->kits_manager->get_current_settings( 'default_generic_fonts' ) . ';',
```
Comment (kobizz) [inline includes/controls/groups/typography.php:93]: Since the comma shouldn't be here when there's no default font, revert the code as it was before

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/controls/groups/typography.php (hunk 3 above)
Comment (matipojo) [reply includes/controls/groups/typography.php:93]: @kobizz  It should be handled by the CSS Parser, something like `{{FONT_VALUE}}` that will automatically add the default font if exist.

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**